src/test/java/neworangehrm/com/opensourcehrm/steps/CommonSteps.java

```diff
@@ -18,16 +18,17 @@
 import neworangehrm.com.opensourcehrm.util.DataUtil;
 
 import static neworangehrm.com.opensourcehrm.pages.LoginPage.*;
+import static neworangehrm.com.opensourcehrm.util.DataUtil.baseUrl;
 
 public class CommonSteps implements En {
 
     DriverUtil driverUtil = new DriverUtil();
     WebDriver driver = driverUtil.getDriverInstance();
 
     public CommonSteps() {
-
+        DataUtil.loadProperties();
```

Comment:
Please move setup related methods to Before annotation.

Reply:
Accepted and Fixed.